### ISSUE TITLE
Is it posible to deploy workspace using windows container instead of linux container

### ISSUE THREAD

**Issue by huonguyenlt** [eclipse/che#23186]
### Summary

I need to deploy workspace that supports dotnet framework. It only run on windows so I prepare a windows host to deploy the workspace.
**Problem**: a workspace pod consits of the following image:
- mcr.microsoft.com/dotnet/framework/sdk:4.8.1
- quay.io/eclipse/che--traefik 
- quay.io/che-incubator/che-code
- quay.io/devfile/project-clone
Only the first image can be pull and run on windows host. Other images are failed to pull because the image operating system is "linux" and it cannot be used on windows.

**Questions**:
- Is there any approach to overcome this issue? 
- Is it possible for you to provide windows version for these required images (che-traefik, che-cide, project-clone)

**Comment by tolusha** [comment]
Hello.

Did you have a chance to look at [devfile.io](https://registry.devfile.io/viewer?search=.net) ?
Maybe they suit your needs.

**Comment by huonguyenlt** [comment]
@tolusha Yes, I checked the devfile.io. They only have dotnet core which run on linux.
My problem is I need dotnet framework, which only runs on windows. If I deploy the workspace pod on windows, the other che's components like che-code, traefik, etc. will not run as they work on linux only.

**Comment by tolusha** [comment]
@svor 
Do you have any ideas?

**Comment by svor** [comment]
afaik there is no alternatives of .NET Framework SDK for linux and i am not sure that che-traefik, che-ide, project-clone are going to be built specifically for Windows
